.gitignore

```diff
@@ -101,7 +101,6 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
-cover/
```

Comment:
How come this was removed?

Reply:
We changed `cover percentage` to `cover`, and the folder cannot be pushed...